examples/axual/22_streams.tf

```diff
@@ -1,17 +1,28 @@
-resource "axual_stream" "gitops_test_stream2" {
-  name = "gitops_test_stream2"
+resource "axual_stream" "logs" {
+  name = "logs"
   key_type = "String"
   value_type = "String"
-  owners = axual_group.gitops_test.id
+  owners = axual_group.developers.id
   retention_policy = "delete"
   properties = { }
 }
 
-resource "axual_stream" "gitops_test_stream3" {
-  name = "gitops_test_stream3"
+resource "axual_stream" "support" {
```

Comment:
Can we add some comments on what this topic is about? Or just set the description?